api-reference/beta/toc.yml

```
@@ -13615,6 +13615,68 @@ items:
               href: api/policyroot-list-rolemanagementpolicyassignments.md
             - name: Get
               href: api/unifiedrolemanagementpolicyassignment-get.md
+          - name: assignment schedule requests
+            href: resources/privilegedaccessgroupassignmentschedulerequest.md
+            items:
+            - name: List
+              href: api/privilegedaccessgroup-list-assignmentschedulerequests.md
+            - name: Create
+              href: api/privilegedaccessgroup-post-assignmentschedulerequests.md
+            - name: Get
+              href: api/privilegedaccessgroupassignmentschedulerequest-get.md
+            - name: Cancel
+              href: api/privilegedaccessgroupassignmentschedulerequest-cancel.md
+            - name: Filter by current user
+              href: api/privilegedaccessgroupassignmentschedulerequest-filterbycurrentuser.md
+          - name: assignment schedules
```

GLOBAL in the TOC: Start with caps
```suggestion
          - name: Assignment schedules
```